### PR TITLE
Adds query execution capabilities to VSCode

### PR DESCRIPTION
**Pull request by ncordon**

### CODE REVIEW

File: packages/vscode-extension/src/webviews/resultPanel.ts [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+export function setAllTabsToLoading(
+  webview: Webview,
+  script: string,
+  ndlCssUri: string,
+): string {
+  const nonce = getNonce();
+
+  return `
```
[the remark this answers is not in the document]
Comment (ncordon) [reply packages/vscode-extension/src/webviews/resultPanel.ts:123]: That seems sensible to me. What order do you want to merge them? This pr first, then the one refactoring the html template?

File: packages/react-codemirror-playground/package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
     "postcss": "^8.4.23",
     "tailwindcss": "^3.3.1",
     "typescript": "^4.9.5",
-    "vite": "^4.2.0"
+    "vite": "^4.2.0",
+    "esbuild": "^0.19.4"
```
Comment (ncordon) [inline packages/react-codemirror-playground/package.json:46]: Without this, I struggle to build the project sometimes in osx:
```
@neo4j-cypher/react-codemirror-playground:build: failed to load config from /Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/vite.config.ts
@neo4j-cypher/react-codemirror-playground:build: error during build:
@neo4j-cypher/react-codemirror-playground:build: Error: The package "@esbuild/darwin-x64" could not be found, and is needed by esbuild.
@neo4j-cypher/react-codemirror-playground:build: 
@neo4j-cypher/react-codemirror-playground:build: If you are installing esbuild with npm, make sure that you don't specify the
@neo4j-cypher/react-codemirror-playground:build: "--no-optional" or "--omit=optional" flags. The "optionalDependencies" feature
@neo4j-cypher/react-codemirror-playground:build: of "package.json" is used by esbuild to install the correct binary executable
@neo4j-cypher/react-codemirror-playground:build: for your current platform.
@neo4j-cypher/react-codemirror-playground:build:     at generateBinPath (/Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/esbuild/lib/main.js:1899:15)
@neo4j-cypher/react-codemirror-playground:build:     at esbuildCommandAndArgs (/Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/esbuild/lib/main.js:1969:33)
@neo4j-cypher/react-codemirror-playground:build:     at ensureServiceIsRunning (/Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/esbuild/lib/main.js:2133:25)
@neo4j-cypher/react-codemirror-playground:build:     at build (/Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/esbuild/lib/main.js:2025:26)
@neo4j-cypher/react-codemirror-playground:build:     at bundleConfigFile (file:///Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:66209:26)
@neo4j-cypher/react-codemirror-playground:build:     at loadConfigFromFile (file:///Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:66185:31)
@neo4j-cypher/react-codemirror-playground:build:     at resolveConfig (file:///Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:65782:34)
@neo4j-cypher/react-codemirror-playground:build:     at build (file:///Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/vite/dist/node/chunks/dep-41cf5ffd.js:47926:26)
@neo4j-cypher/react-codemirror-playground:build:     at CAC.<anonymous> (file:///Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemirror-playground/node_modules/vite/dist/node/cli.js:842:15)
@neo4j-cypher/react-codemirror-playground:build: npm ERR! Lifecycle script `build` failed with error: 
@neo4j-cypher/react-codemirror-playground:build: npm ERR! Error: command failed 
@neo4j-cypher/react-codemirror-playground:build: npm ERR!   in workspace: @neo4j-cypher/react-codemirror-playground@2.0.0-next.9 
@neo4j-cypher/react-codemirror-playground:build: npm ERR!   at location: /Users/ncordon/neo4j/intellisense/cypher-lsp/packages/react-codemir
```

File: packages/vscode-extension/src/webviews/resultPanel.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (daveajrussell) [reply packages/vscode-extension/src/webviews/resultPanel.ts]: shall we merge mine later? 👍

**Review by daveajrussell — APPROVED** [review]
Looks great, working as expected 👍

**Pull request merged**